packages/devtools_app/lib/src/charts/chart_trace.dart

```diff
@@ -33,13 +44,23 @@ class PaintCharacteristics {
   /// Will assert if new data point is more than max specified.
   double fixedMaxY;
 
+  /// Primary color.
   Color color;
 
+  /// Used for DataCount, if count > 1 use this color.
+  Color colorAggregate;
```

Comment:
Nit: maybe rewrite as 
/// Color to use if count > 1.
///
/// See [DataCount.count].

Reply:
Done.
```
  /// Color to use if count > 1.
  ///
  /// See [DataAggregate.count].

```